adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -865,22 +971,18 @@
             loginIframe.iframe = iframe;
 
             iframe.onload = function() {
-                var realmUrl = getRealmUrl();
-                if (realmUrl.charAt(0) === '/') {
+                var authUrl = kc.authorize();
```

Comment:
this should be "kc.endpoints.authorize()"

Reply:
Oh yes, sure. Sorry about that. `checkSessionIframe` was not used in my main use-case, so I didn't spot this error.

Reply:
Fixed in commit https://github.com/keycloak/keycloak/pull/4978/commits/7274e8d9dba2c75a2ff3363a344fa85a3e4dcb15